packages/shared-business/src/locales/en.json

```diff
@@ -257,6 +257,8 @@
   "supportingDoc.documentTypes": "Supported documents: .pdf, .png, .jpg\nMax: {maxSizeMB} MB",
   "supportingDoc.downloadTemplate": "Download template",
   "supportingDoc.errorUpload": "Failed to upload file",
+  "supportingDoc.generalAssemblyMinutes": "General Assembly Minutes",
```

Comment:
Wrong casing, should be `General assembly minutes` to be coherent with the others (I also think that `UBO Declaration` should be `UBO declaration`, but that's another topic)